src/editors/ConsumptionLeisureEditor.jsx

```diff
@@ -30,31 +32,33 @@ export default class ConsumptionLeisureEditor extends React.Component {
                     <React.Fragment>
                         <h2>Slope</h2>
                         <RangeEditor
-                            itemlabel="Horizontal intercept value <strong>T</strong>"
+                            itemlabel={["Horizontal intercept value: ", <strong>T</strong>]}
                             id="gA1"
                             dataId="gA1"
                             value={this.props.gA1}
                             min={0}
                             max={9}
                             handler={handleFormUpdate.bind(this)} />
                         <RangeEditor
-                            itemlabel="Real Wage <strong>w</strong>"
+                            itemlabel={["Real Wage: ", <strong>w</strong>]}
                             id="gA2"
                             dataId="gA2"
                             value={this.props.gA2}
                             min={0.01}
                             max={5}
                             handler={handleFormUpdate.bind(this)} />
+                        {this.props.gType === 15 && (
+                            <RangeEditor
+                                itemlabel={["Rel. Preference: ", <strong>α</strong>]}
```

Comment:
Nice workaround for displaying HTML tags properly in the RangeEditor. Another way to go about this (and what would be my first instinct) would be to use [dangerouslySetInnerHTML](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) in the RangeEditor component and keep `itemlabel` as a string containing html, but this array scheme seems to work fine as well.

Reply:
I looked into dangerouslySetInnerHTML and I was STRONGLY encouraged not to. Apparently it creates a security risk for certain kinds of code insertion.

Reply:
We are using this elsewhere in the application, e.g. rendering instructions in `src/GraphViewer.jsx`, where we're actually using it on a user-provided string, where there is a risk of code insertion - but this is an instructor form, and I trust Tom.

There is no real danger to using it in this scenario where we are just trying to render a pre-defined string... that I can think of, at least. I believe React is just being extra cautious by naming it "dangerous" - according to their own docs:
> In general, setting HTML from code is risky because it’s easy to inadvertently expose your users to a [cross-site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) attack.

That's definitely true - and I'd argue we know what we're doing in this scenario, so it's fine to use!